examples/mobile/UIComponents/components/progress/progress-bar/progressbar.css

```diff
@@ -0,0 +1,4 @@
+.ui-progress {
```

Comment:
Which ui-progress or ui-progressbar is correct?

Reply:
According to [developers.tizen.org](https://developer.tizen.org/development/api-references/web-application?redirect=/dev-guide/5.0.0/org.tizen.web.apireference/html/ui_fw_api/Mobile_UIComponents/mobile_Progress.htm#default-selectors0.9354383451864123) the proper way is:
``` html
<div class="ui-progress" data-type="bar"></div>
```

Reply:
as we discussed we move this to TAU